frontend/lib/verification/view/verification_result_card.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+
+import 'content_card.dart';
+
+class VerificationResultCard extends StatelessWidget {
+  final Widget child;
+  final String title;
+  final bool isPositive;
+
+  VerificationResultCard({
+    Key key,
+    this.child,
+    this.title,
+    @required this.isPositive
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    var icon = isPositive ? Icons.verified_user : Icons.error;
+    var color = isPositive ? Colors.green : Colors.red;
+    return ContentCard(
+        borderSide: isPositive
+            ? const BorderSide(width: 1.0, color: Colors.black12)
+            : const BorderSide(color: Colors.red, width: 4.0),
+        child: Column(
+        children: [
+          RichText( // TODO feels strange to use RichText. Why not Row?
```

Comment:
tbh, not 100% sure but that's what richtext is doing, I just took that from the docs 😆

Reply:
Ah, okay 😄:+1: